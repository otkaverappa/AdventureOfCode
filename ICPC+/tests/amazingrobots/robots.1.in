1 1
X
0
5 4
##.#
#..#
#.##
#.X#
####
0
