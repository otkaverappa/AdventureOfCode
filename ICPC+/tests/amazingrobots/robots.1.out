6
W
N
N
E
N
N
